# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: architecture/powerpc64/package/*/0700-ps3vram-mod-ps3fb.patch
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- linux-6.4/arch/powerpc/platforms/ps3/Kconfig.vanilla	2023-08-17 13:33:39.151911643 +0200
+++ linux-6.4/arch/powerpc/platforms/ps3/Kconfig	2023-08-17 13:34:02.434911004 +0200
@@ -180,7 +180,7 @@
 
 config PS3_VRAM
 	tristate "PS3 Video RAM Storage Driver"
-	depends on FB_PS3=y && BLOCK && m
+	depends on FB_PS3 && BLOCK && m
 	help
 	  This driver allows you to use excess PS3 video RAM as volatile
 	  storage or system swap.
